web/static.go

```diff
@@ -81,7 +81,11 @@ func staticFilesHandler(handler http.Handler) http.Handler {
 		//wrap our ResponseWriter with our no-cache 404-handler
 		w = &notFoundNoCacheResponseWriter{ResponseWriter: w}
 
-		w.Header().Set("Cache-Control", "max-age=31556926, public")
+		if path.Base(r.URL.Path) == "remote_entry.js" {
+			w.Header().Set("Cache-Control", "no-cache, max-age=0")
+		} else {
+			w.Header().Set("Cache-Control", "max-age=31556926, public")
+		}
```

Comment:
Maybe refactor to have a `shouldAvoidCachingAsset` or something like that? It's just one case atm so not a big deal

Reply:
Yeah, I think let's leave it as-is for now. We can give it a nicer name if we end up needing this elsewhere, but hopefully that won't be the case.